src/content/en/2020/http2.md

```diff
@@ -8,9 +7,9 @@ authors: [dotjs, rmarx, MikeBishop]
 reviewers: [LPardue, bazzadp, ibnesayeed]
```

Comment:
The [doc](https://docs.google.com/document/d/1ybIBEtM-C2e-pEZMs_bp39mtEs3u6s8LHOz6Es8QUGA/edit#heading=h.xleqpbggrlck) lists @LPardue as an author, not reviewer.

Which is correct? @dotjs @LPardue

Reply:
I wanted to author but the other folks did such a good job all I could do is review 😀